src/ext-test/resources/ridehailing/uber-price-estimates.json

```diff
@@ -0,0 +1,92 @@
+{
+  "prices": [
+    {
+      "localized_display_name": "POOL",
+      "distance": 6.17,
```

Comment:
In the OTP1 implementation, estimated duration and distance from the ride hailing company were included in the ride hailing esimates used by the UI, but we can derive them from the leg data, so that shouldn't be a problem.